gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -152,24 +152,40 @@ services:
     enabled: true
     tryAvoidDuplicateNotification: false
 
-  # Rules about certificate expiry notifications.
-  # Require the platform notifier service.
-  certificate:
-    enabled: true
-    # frequency on which the notifier mechanism will test
-    # if new notifications need to be send
-    # default: 0 0 5 * * * (every day at 5am)
-    cronExpression: 0 0 5 * * *
-    # send notification if certificate is going to expire in less than 20 days, then 15 days before...
-    expiryThresholds: 20,15,10,5,1
-    # Subject of the email send by the email notifier
-    expiryEmailSubject: Certificate will expire soon
+    ## Rules about certificate expiry notifications. Require the platform notifier service.
+    #certificate:
+      #enabled: true
+      ## frequency on which the notifier mechanism will test
+      ## if new notifications need to be send
+      ## default: 0 0 5 * * * (every day at 5am)
+      #cronExpression: 0 * * * * *
+      ## send notification if certificate is going to expire in less than 20 days, then 15 days before...
+      #expiryThresholds: 400,15,10,5,1
+      ## Subject of the email send by the email notifier
+      #expiryEmailSubject: Certificate will expire soon
+
+    # Rules about client's secrets expiry notifications. Require the platform notifier service.
+    client-secret:
+      enabled: true
+      # frequency on which the notifier mechanism will test
+      # if new notifications need to be send
+      # default: 0 0 5 * * * (every day at 5am)
+      cronExpression: 30 * * * * *
+      # send notification if client secret is going to expire in less than 20 days, then 15 days before...
+      expiryThresholds: 20,15,10,5,1
+      expiryEmailSubject: Client secret will expire soon
 
 license:
   expire-notification:
     enabled: true
 
 notifiers:
+  kafka:
+    enabled: true
+    bootstrapServers: localhost:9092
+    topic: test
+    acks: 1
```

Comment:
todo: as Kafka is a EE plugin, please remove these settings. The settings information will have to be provided in the documenation